src/components/overview/select/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { StyledSelect } from './styled';
+import { MenuItem, SelectChangeEvent } from '@mui/material';
+
+interface SelectProps {
+  value: string;
+  onChange: (event: SelectChangeEvent<string>) => void;
+  options: { [key: string]: string };
+}
+const CustomSelect: React.FC<SelectProps> = ({ value, onChange, options }) => {
+  return (
+    <StyledSelect value={value} onChange={(e) => onChange(e as SelectChangeEvent<string>)}>
+      {Object.entries(options).map(([value, label]) => (
```

Comment:
Why you decided to use  Object format instead of Array for options?
What if I want to use this `CustomSelect` to render a list or tickets inside, for example. I always need to create Object for it?

Reply:
fix: changed to array format